src/interfaces/PlayerOptions.ts

```diff
@@ -4,6 +4,7 @@ import type {
   IOSCategoryMode,
   IOSCategoryOptions,
 } from '../constants';
+import { AndroidAudioUsageType } from '../constants/AndroidAudioUsageType';
```

Comment:
Please export `AndroidAudioUsageType` from `constants/index.ts` so it can be imported in a consistent manner to the others.